apps/crn-server/test/fixtures/users.fixtures.ts

```diff
@@ -17,7 +17,7 @@ import { EventBridgeEvent } from 'aws-lambda';
 import { createEventBridgeEventMock } from '../helpers/events';
 
 export const fetchUserResponseDataObject = (): UserDataObject => ({
-  avatarUrl: `${baseUrl}/api/assets/${appName}/squidex-asset-id`,
+  avatarUrl: `${baseUrl}/api/assets/${appName}/contentful-asset-id`,
```

Comment:
Does this test fixture need to use `baseUrl` and `appName`, why can it not just be fake? eg `https://test-avatar-asset-id`